src/test/run-pass/borrowck/borrowck-borrow-of-projection-kills-other-borrows-issue-62007.rs

```diff
@@ -0,0 +1,21 @@
+// run-pass
```

Comment:
A comment would be good :)

Reply:
(to clarify: it tends to be a good idea to not rely solely on the filename for a test for the description of what its testing. You can just transcribe part of the description from #62007 to explain it.)